python/chronos/test/bigdl/chronos/data/experimental/test_xshardstsdataset.py

```diff
@@ -192,6 +192,20 @@ def test_xshardstsdataset_roll_multiple_id(self):
         x = np.concatenate([collected_numpy[i]['x'] for i in range(len(collected_numpy))], axis=0)
         assert x.shape == ((50-lookback-horizon+1)*2, lookback, 2)
 
+    def test_xshardstsdataset_impute(self):
+        for val in ["last", "const", "linear"]:
+            shards_tmp = read_csv(os.path.join(self.resource_path, "impute_test.csv"))
```

Comment:
I think we can generate the csv and dump the csv only for temp use. We don't need to upload the `impute_test.csv`